testeng/jobs/repoHealthReport.groovy

```diff
@@ -0,0 +1,87 @@
+package testeng
+
+def pytest_repo_health_gitURL = 'git@github.com:edx/pytest-repo-health.git'
+def edx_repo_health_gitURL = 'git@github.com:edx/edx-repo-health.git'
+def destination_repo_health_gitURL = "git@github.com:edx/repo-tools-data.git"
+def githubUserReviewers = []
+def githubTeamReviewers = ['platform-core', 'arch-bom']
+
+
+job('repo-health-report') {
+
+    description('Generate a report listing repository structure standard compliance accross edX repos')
+    parameters {
+        stringParam('GITHUB_REPO_URL', 'https://github.com/edx/edx-platform',
+                    'Github repo url on which to run pytest-repo-health checks')
+    }
+    concurrentBuild(false)
+    environmentVariables(
+            PR_USER_REVIEWERS: githubUserReviewers.join(","),
+            PR_TEAM_REVIEWERS: githubTeamReviewers.join(",")
+        )
+    multiscm {
+        git {
+            remote {
+                url(pytest_repo_health_gitURL)
+            }
+            branch('master')
+            browser()
+            extensions {
+                cleanAfterCheckout()
+                relativeTargetDirectory('pytest-repo-health')
+            }
+        }
+        git {
+            remote {
+                url(edx_repo_health_gitURL )
+            }
+            branch('master')
+            browser()
+            extensions {
+                cleanAfterCheckout()
+                relativeTargetDirectory('edx-repo-health')
+            }
+        }
+        git {
+                remote {
+                    url('https://github.com/edx/testeng-ci.git')
+                }
+                branch('*/msingh/repo_health')
+                extensions {
+                    cleanBeforeCheckout()
+                    relativeTargetDirectory('testeng-ci')
+                }
+            }
+        git {
+            remote {
+                credentials('jenkins-worker')
+                url('${GITHUB_REPO_URL}')
```

Comment:
Did you test this? I don't think you need curly braces here. There are many examples in the repo. For example https://github.com/edx/jenkins-job-dsl/blob/master/devops/jobs/MongoAgentsUpdate.groovy#L39